lore/estimators/keras.py

```diff
@@ -177,63 +177,99 @@ def build_inputs(self):
             if hasattr(encoder, 'sequence_length'):
                 for i in range(encoder.sequence_length):
                     inputs[encoder.sequence_name(i)] = Input(shape=(1,), name=encoder.sequence_name(i))
+                    if encoder.twin:
+                        inputs[encoder.sequence_name(i, suffix='_twin')] = Input(shape=(1,), name=encoder.sequence_name(i, suffix='_twin'))
```

Comment:
Should we have a const in `lore.encoders.TWIN = '_twin'` or is that overkill? Another alternative would be to add `twin_sequence_name()`

Reply:
Done